scalr/README.md

```diff
@@ -0,0 +1,71 @@
+# Scalr Integration
+
+## Overview
+
+This integration collects metrics from Scalr.
+
+## Setup
+The Scalr integration is not included in the [Datadog Agent][2] package, so you need to install it.
+
+### Installation
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Scalr integration on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
```

Comment:
Is there a more concrete version support that we can define here besides "6.21 and later, 7.21 and later", like just "v6.21 and later"?

```suggestion
For the Datadog Agent v7.21 or v6.21 and later, follow these instructions to install the Scalr integration on your host. See [Use Community Integrations][3] to install it with the Docker Agent or earlier versions of the Datadog Agent.
```